spring-data-mongodb/src/main/java/org/springframework/data/mongodb/core/MongoExceptionTranslator.java

```diff
@@ -49,21 +52,28 @@
  * @author Oliver Gierke
  * @author Michal Vich
  * @author Christoph Strobl
+ * @author Brice Vandeputte
  */
 public class MongoExceptionTranslator implements PersistenceExceptionTranslator {
+	/**
+	 * translation matrix definitions:
+	 * - adding {@code ClassShortName} will associate a given exception class to a translated group
+	 * - adding {@code ClassShortName$children} will associate a given exception direct children to a translated group
+	 */
 
 	private static final Set<String> DUPLICATE_KEY_EXCEPTIONS = new HashSet<>(
-			Arrays.asList("MongoException.DuplicateKey", "DuplicateKeyException"));
+			asList("MongoException.DuplicateKey", "DuplicateKeyException"));
```

Comment:
Nit: We typically don't use static imports to retain the context.

Reply:
sure I would revert that on next upd